src/SimpleAzure.Storage.HybridQueue/HybridQueue.cs

```diff
@@ -7,21 +7,12 @@
 
 namespace WorldDomination.SimpleAzure.Storage.HybridQueues;
 
-public class HybridQueue : IHybridQueue
+public sealed class HybridQueue(QueueClient queueClient, BlobContainerClient blobContainerClient, ILogger<HybridQueue> logger) : IHybridQueue
 {
-    private readonly QueueClient _queueClient;
-    private readonly BlobContainerClient _blobContainerClient;
-    private readonly ILogger<HybridQueue> _logger;
+    private readonly QueueClient _queueClient = queueClient;
+    private readonly BlobContainerClient _blobContainerClient = blobContainerClient;
+    private readonly ILogger<HybridQueue> _logger = logger;
 
-    /// <inheritdoc />

```

Comment:
Because the IDE automatically shows inherited doc comments. If you were generating a DocFX page from this, they might be needed, not sure. In my opinion they're just noise.

Reply:
> Because the IDE automatically shows inherited doc comments.

wait what?! really?

Also -> _Visual Studio_ IDE only? there are 3 in the market ...
- VS
- VSC
- R

Reply:
![image](https://github.com/PureKrome/WorldDomination.SimpleAzure.Storage.HybridQueues/assets/895629/a5c16168-4757-4fbe-afe6-7c7d7cf5c02a)
![image](https://github.com/PureKrome/WorldDomination.SimpleAzure.Storage.HybridQueues/assets/895629/2fa91089-d4b4-4f3f-b55b-1792b1c03294)
![image](https://github.com/PureKrome/WorldDomination.SimpleAzure.Storage.HybridQueues/assets/895629/d9561569-c799-4cfd-8c05-56df00871ae1)


The ctor one doesn't work as there's no ctor on the interface to inherit from.

Reply:
I think he probably wanted to know if Rider shows the doc comments when the inheritdoc isn't there, to know whether its okay to remove it 😛

Reply:
![image](https://github.com/PureKrome/WorldDomination.SimpleAzure.Storage.HybridQueues/assets/895629/63e0d29e-3d18-4ce7-b57f-8c5d41feaede)

Reply:
Perfect! thanks gents!